modules/network-observability-operator-install.adoc

```diff
@@ -22,7 +22,7 @@ The actual memory consumption of the Operator depends on your cluster size and t
 
 [NOTE]
 ====
-This documentation assumes that your `LokiStack` instance name is `loki`. Using a different name requires additional configuration.
+This documentation assumes that your `LokiStack` instance name is `loki`. Using a different name requires additional configuration. Additionally, this installation example uses the `netobserv` namespace, which is used across all components. You can optionally use a different namespace.
```

Comment:
I think we can remove this part "This documentation assumes that your `LokiStack` instance name is `loki`. Using a different name requires additional configuration."
Because now if they use a different LokiStack name, they just need to adapt the LokiStack reference and that's it, there shouldn't be other assumptions related to that name in the doc I think